docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -29,7 +29,8 @@ The command syntax is as follows:
    scylla sstable <operation> <path to SStable>
 
 
-You can specify more than one SStable.
+You can specify more than one SSTable. Additionally, the path to SSTable can point to an S3 fully qualified path in the form of s3://bucket-name/prefix/of/your/sstable/sstable-TOC.txt. To use this feature, you need to have AWS credentials set up in your environment. For more information, see :ref:`Configuring AWS S3 access <aws-s3-configuration>`. Additionally, you must use the ``--scylla-yaml-file`` parameter for the tool to obtain S3 endpoint configuration.
```

Comment:
You don't have to use `--scylla-yaml-file` for the tool to successfully load the yaml file. The tool will look at reasonable default locations. I would rephrase this to say that the user should make sure the tool is able to load the correct scylla yaml file.